pkg/cloud/aws/actuators/machine/security_groups.go

```diff
@@ -31,7 +31,7 @@ const (
 	// annotation which tracks the SecurityGroups that the machine actuator is
 	// responsible for. These are the SecurityGroups that have been handled by
 	// the AdditionalSecurityGroups in the Machine Provider Config.
-	SecurityGroupsLastAppliedAnnotation = "sigs.k8s.io/cluster-api-provider-aws/last-applied/security-groups"
+	SecurityGroupsLastAppliedAnnotation = "sigs.k8s.io/cluster-api-provider-aws-last-applied-security-groups"
```

Comment:
Could we link the [annotation rules](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/) in the comment above?

Reply:
done!